operations/security_and_authentication/Change_credentials_on_ServerTech_PDUs.md

```diff
@@ -0,0 +1,133 @@
+# Change credentials on ServerTech PDUs
+This procedure changes password used by the `admn` user on ServerTech PDUs. Either a single PDU can be updated to a new credential, or update all ServerTech PDUs in the system to the same global credentials.
+
+**Note:** This procedure does not update the default credentials that RTS uses for new ServerTech PDUs added to a system. To change the default credentials follow the [Update default ServerTech PDU credentials used by the Redfish Translation Service](Update_default_ServerTech_PDU_credentials_used_by_the_Redfish_Translation_Service.md) procedure.
```

Comment:
Add comma after credentials in second sentence

Reply:
Should be resolved fixed now.